widgetssdk/src/main/java/com/glia/widgets/di/GliaCoreImpl.java

```diff
@@ -144,7 +144,7 @@ public void unsubscribeFromQueueUpdates(Consumer<GliaException> onError,
     @Override
     public void subscribeToQueueTicketUpdates(@NonNull String ticketId,
                                               RequestCallback<QueueTicket> callback) {
-        Glia.subscribeToQueueTicketUpdates(ticketId, callback);
+//        Glia.subscribeToQueueTicketUpdates(ticketId, callback);
```

Comment:
Could you please explain the reason for disabling this?

Reply:
Left over from debuging. Fixed ccca28d